presentation/frontend/src/test/favorite-properties/FavoritePropertiesManager.test.ts

```diff
@@ -61,59 +59,55 @@ describe("FavoritePropertiesManager", () => {
   describe("initializeConnection", () => {
 
-   // SWB
-    it("loads project and iModel scopes", async () => {
+    it("loads iTwin and iModel scopes", async () => {
       await manager.initializeConnection(imodelMock.object);
       storageMock.verify(async (x) => x.loadProperties(undefined, undefined), moq.Times.once());
-      storageMock.verify(async (x) => x.loadProperties(projectId, imodelId), moq.Times.once());
-      storageMock.verify(async (x) => x.loadProperties(projectId, undefined), moq.Times.once());
+      storageMock.verify(async (x) => x.loadProperties(itwinId, imodelId), moq.Times.once());
+      storageMock.verify(async (x) => x.loadProperties(itwinId, undefined), moq.Times.once());
     });
 
-   // SWB
-    it("loads iModel scope when project scope is already loaded", async () => {
+    it("loads iModel scope when iTwin scope is already loaded", async () => {
       await manager.initializeConnection(imodelMock.object);
 
       const imodelId2 = "imodel-id-2";
       imodelMock.reset();
       imodelMock.setup((x) => x.iModelId).returns(() => imodelId2);
-      imodelMock.setup((x) => x.iTwinId).returns(() => projectId);
+      imodelMock.setup((x) => x.iTwinId).returns(() => itwinId);
       await manager.initializeConnection(imodelMock.object);
 
       storageMock.verify(async (x) => x.loadProperties(undefined, undefined), moq.Times.once());
-      storageMock.verify(async (x) => x.loadProperties(projectId, imodelId2), moq.Times.once());
-      storageMock.verify(async (x) => x.loadProperties(projectId, undefined), moq.Times.once());
+      storageMock.verify(async (x) => x.loadProperties(itwinId, imodelId2), moq.Times.once());
+      storageMock.verify(async (x) => x.loadProperties(itwinId, undefined), moq.Times.once());
     });
 
     it("does not load iModel scope when iModel scope is already loaded", async () => {
       await manager.initializeConnection(imodelMock.object);
       await manager.initializeConnection(imodelMock.object);
 
       storageMock.verify(async (x) => x.loadProperties(undefined, undefined), moq.Times.once());
-      storageMock.verify(async (x) => x.loadProperties(projectId, imodelId), moq.Times.once());
-      storageMock.verify(async (x) => x.loadProperties(projectId, undefined), moq.Times.once());
+      storageMock.verify(async (x) => x.loadProperties(itwinId, imodelId), moq.Times.once());
+      storageMock.verify(async (x) => x.loadProperties(itwinId, undefined), moq.Times.once());
     });
 
     it("removes non-favorited property order information", async () => {
       const globalField = createTestPropertiesContentField({ properties: [{ property: createTestPropertyInfo({ name: "global" }) }] });
-      // SWB
-      const projectField = createTestPropertiesContentField({ properties: [{ property: createTestPropertyInfo({ name: "project" }) }] });
+      const iTwinField = createTestPropertiesContentField({ properties: [{ property: createTestPropertyInfo({ name: "itwin" }) }] });

```

Comment:
Breaking convention with itwin and lowercase T

Reply:
Fixed to iTwin, since this is confined to a single test it does not have any side effects.